packages/plugins/solc/package.json

```diff
@@ -36,30 +36,59 @@
     "_typecheck": "npm run solo -- typecheck",
     "ci": "npm run qa",
     "clean": "npm run reset",
-    "lint": "eslint src/",
+    "lint": "eslint src/ test/",
     "qa": "npm-run-all lint _typecheck _build",
     "reset": "npx rimraf dist embark-*.tgz package",
-    "solo": "embark-solo"
+    "solo": "embark-solo",
+    "test": "jest"
   },
   "eslintConfig": {
-    "extends": "../../../.eslintrc.json"
+    "extends": [
+      "../../../.eslintrc.json",
+      "plugin:jest/recommended",
+      "plugin:jest/style"
+    ]
   },
   "dependencies": {
     "@babel/runtime-corejs3": "7.8.4",
     "async": "3.2.0",
-    "core-js": "3.4.3",
-    "semver": "5.6.0",
-    "shelljs": "0.8.3"
+    "core-js": "3.6.4",
+    "global": "4.4.0",
+    "semver": "7.1.3",
+    "shelljs": "0.8.3",
+    "solc": "0.6.3"
```

Comment:
I saw `solc@0.6.3`  was added to `yarn.lock` and was curious why it's added as a dependency here?

Reply:
Good catch! That shouldn't be needed as we're using `solc` not `solcjs`. Will update.